### ISSUE TITLE
Buttons just disappeared?

### ISSUE THREAD

**Issue by CTegtmeyer** [spawaskar-cora/cora-docs#370]
I was entering information into CoRA and all of the buttons just disappeared. I've relogged, and cleared my cache and it still looks the same. They're still there if I scroll over them, but they've disappeared from the screen and I am not sure why. (I'd attach a screen capture, but of course, I cannot because I am on Edge). Is anyone else having this problem?

**Comment by jjlynch2** [comment]
I also see this issue. The buttons (icons) on the left-side navigation are not showing.

**Comment by SachinPawaskarUNO** [comment]
@CTegtmeyer @jjlynch2 @fedamann, My best guess right now is that the something was upgraded on your local machines for example the edge browser, or OS patch. Maybe a new version was pushed out or something, Because the logs show that several users have used CoRA since 3:00 pm CST and have been doing lots of different activities on CoRA. Please see if there are version differences with others in your Office for whom it's working fine.
@CTegtmeyer and @jjlynch2 Can you please check and see who it is working for and see if there are different versions of edge browser and/or different versions of OS patch level?
 
@fedamann, I'm a little concerned though because if this is a OS or Browser update that DPAA IT pushed out then eventually every machine in DPAA might get it. So we need to investigate this issue ASAP.

**Comment by fedamann** [comment]
@CTegtmeyer and @jjlynch2 what is your winver on the machine where this happened?

**Comment by jjlynch2** [comment]
@fedamann @SachinPawaskarUNO This happened on the Windows 10 laptop I've been using. I will check tomorrow for the browser and OS patch version. I don't recall seeing an update notification, but maybe it was pushed overnight.

**Comment by nyukyi** [comment]
I have the same problem too, and I have winver 1607.  The buttons came back for about 3 seconds after @CTegtmeyer and I discovered this problem, but it suddenly logged me out (maybe @SachinPawaskarUNO was fixing something?).  When I logged back in, the buttons are gone again.

**Comment by cbrown311** [comment]
I didn't have the buttons when working today either. I was in Edge, winver 1607. I can still do everything I need to but basically have to remember what was where because ico d are gone.

**Comment by jjlynch2** [comment]
@SachinPawaskarUNO I can see the icons from my home machine (chromium on Linux and edge on win10 1803).

Also, I should note that I experienced the logout that @nyukyi mentioned during work hours. It appeared that my session was  terminated.

**Comment by SachinPawaskarUNO** [comment]
@cbrown311 Thank you for the quick response. That is good to know.
@fedamann Next step would be to see if you can access this on Edge from outside of DPAA network. BTW I just tested this with using Windows 10 and IE Edge Browser see screenshot below. Seems to be working fine (versions might be different)

![image](https://user-images.githubusercontent.com/19599773/52319351-d0ff5500-298e-11e9-920e-72f80d1aa014.png)

My suspicion is that DPAA IT might have pushed new version of OS or Browser or some new network rules that are preventing/blocking some font/icon styling css or js files. I'll verify this when I get there this Thursday for our SCRUM.

**Comment by clegarde** [comment]
I was using CoRA on Google Chrome and did not have any issues. I don't have Edge on my Mac to test. @SachinPawaskarUNO

**Comment by SachinPawaskarUNO** [comment]
@jjlynch2 Thanks for verifying. That is good news.
The session was terminated issue is a know issue because of memory resources limitations. The 2 are not related. See memory threshold being crossed in screenshot below several times today.
![image](https://user-images.githubusercontent.com/19599773/52319519-cb563f00-298f-11e9-9491-0ae51d97ee86.png)

**Comment by SachinPawaskarUNO** [comment]
@clegarde Thanks you for verifying/testing that out for me.

**Comment by fedamann** [comment]
Chrome and Edge 1803 provide full function on my home machines.

**Comment by CTegtmeyer** [comment]
@SachinPawaskarUNO @fedamann I am running CoRA on edge from my work laptop, windows version 1607 OS build 14393.2724.

Like others, when I run it from my personal computer, on chrome, the buttons are there.

I don't recall them pushing an update to my computer at all. I was literally in the middle of working on CoRA when the buttons disappeared, so if they did, it was a background update.

**Comment by SachinPawaskarUNO** [comment]
@CTegtmeyer, Thank you for confirming the OS version, This was released January 8th, 2019 by Microsoft and an update was probably silently pushed yesterday. See below from Micorsoft's support website. I have also attached the
[support link](https://support.microsoft.com/en-us/help/4480961/windows-10-update-kb4480961) here for convenience.

January 8, 2019—KB4480961 (**OS Build 14393.2724**)
**Applies to: Windows 10, version 1607, Windows Server 2016**
Release Date:January 8, 2019
**Version:OS Build 14393.2724**

**Windows 10, version 1607, reached end of service on April 10, 2018**. Devices running Windows 10 Home or Pro editions will no longer receive monthly security and quality updates that contain protection from the latest security threats. To continue receiving security and quality updates, Microsoft recommends updating to the latest version of Windows 10.
